src/main/java/mate/jdbc/dao/ManufacturerDao.java

```diff
@@ -0,0 +1,16 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import mate.jdbc.model.Manufacturer;
+
+public interface ManufacturerDao {
+    Manufacturer create(Manufacturer manufacturer);
+
+    Manufacturer get(Long id);
+
+    List<Manufacturer> getAll() throws RuntimeException;
```

Comment:
```suggestion
    List<Manufacturer> getAll();
```
This is unnecessary.